glusterd2/xlator/actors.go

```diff
@@ -6,24 +6,36 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var optionActors = make(map[string]OptionActor)
+//OptionActors in-memory to store registered xlator type and OptionActor
```

Comment:
Do not export this. Write a function that returns the map.

Reply:
done